examples/dt/dcn/service-values.yaml.j2

```diff
@@ -0,0 +1,160 @@
+  cinderBackup:
+    replicas: 3
+    customServiceConfig: |
+      [DEFAULT]
+      backup_driver = cinder.backup.drivers.ceph.CephBackupDriver
+      backup_ceph_pool = backups
+      backup_ceph_user = openstack
+  cinderVolumes:
+{% for _ceph in _ceph_vars_list %}
+    {{ _ceph.cifmw_ceph_client_cluster }}:
+      customServiceConfig: |
+        [DEFAULT]
+        enabled_backends = ceph
+{% if 'ceph' not in _ceph.cifmw_ceph_client_cluster %}
+        glance_api_servers = https://glance-{{ _ceph.cifmw_ceph_client_cluster }}-internal.openstack.svc:9292
+{% endif %}
+        [ceph]
+        volume_backend_name = ceph
+        volume_driver = cinder.volume.drivers.rbd.RBDDriver
+        rbd_ceph_conf = /etc/ceph/{{ _ceph.cifmw_ceph_client_cluster }}.conf
+        rbd_user = openstack
+        rbd_pool = volumes
+        rbd_flatten_volume_from_snapshot = False
+        rbd_secret_uuid = {{ _ceph.cifmw_ceph_client_fsid }}
+        rbd_cluster_name = {{ _ceph.cifmw_ceph_client_cluster }}
+        backend_availability_zone = {{ _ceph.cifmw_ceph_client_cluster }}
+{% endfor %}
+  glance:
+    customServiceConfig: |
+        [DEFAULT]
+        enabled_backends = default_backend:rbd
+        [glance_store]
+        default_backend = default_backend
+        [default_backend]
+        rbd_store_ceph_conf = /etc/ceph/ceph.conf
+        store_description = "RBD backend"
+        rbd_store_pool = images
+        rbd_store_user = openstack
+        rbd_thin_provisioning = True
+    glanceAPIs:
+{% set backends = [] %}
+{% for _ceph in _ceph_vars_list %}
+{% if _ceph.cifmw_ceph_client_cluster not in backends %}
+{% set _ = backends.append(_ceph.cifmw_ceph_client_cluster + ':rbd') %}
+{% endif %}
+{% endfor %}
+{% for _ceph in _ceph_vars_list %}
+{% if 'ceph' in _ceph.cifmw_ceph_client_cluster %}
+      default:
+{% else %}
+      {{ _ceph.cifmw_ceph_client_cluster }}:
+{% endif %}
+        customServiceConfig: |
+          [DEFAULT]
+          enabled_import_methods = [web-download,copy-image,glance-direct]
+          enabled_backends = {{ backends | join(',') }}
+          [glance_store]
+          default_backend = {{ _ceph.cifmw_ceph_client_cluster }}
+          [{{ _ceph.cifmw_ceph_client_cluster }}]
+          rbd_store_ceph_conf = /etc/ceph/{{ _ceph.cifmw_ceph_client_cluster }}.conf
+          store_description = "{{ _ceph.cifmw_ceph_client_cluster }} RBD backend"
+          rbd_store_pool = images
+          rbd_store_user = openstack
+          rbd_thin_provisioning = True
+{% for _ceph_az in _ceph_vars_list %}
+{% if _ceph_az.cifmw_ceph_client_cluster != _ceph.cifmw_ceph_client_cluster %}
+          [{{ _ceph_az.cifmw_ceph_client_cluster }}]
+          rbd_store_ceph_conf = /etc/ceph/{{ _ceph_az.cifmw_ceph_client_cluster }}.conf
+          store_description = "{{ _ceph_az.cifmw_ceph_client_cluster }} RBD backend"
+          rbd_store_pool = images
+          rbd_store_user = openstack
+          rbd_thin_provisioning = True
```

Comment:
Could we add lines (after this one) to enable automatic image format conversion to raw format? It'd make it easier to import the image in raw format to all glance stores without a need to convert the cirros image from qcow2 to raw format with qemu-img tool (which is not available in opentstackclient pod and It makes it more comfortable if once wanna execute for example tempest test since tempestconf has no ability to create an image in multiple glance stores). It makes sense anyway since we use ceph storage as the backend.
The config lines to enable it would be:

```
            [image_import_opts]
            image_import_plugins = ['image_conversion']
            [image_conversion]
            output_format = raw

```

Reply:
done